tests/client/congressvoting_test.go

```diff
@@ -14,11 +14,14 @@ import (
 	"boscoin.io/sebak/lib/transaction/operation"
 	"github.com/stellar/go/keypair"
 	"github.com/stretchr/testify/require"
+	"boscoin.io/sebak/lib/common"
 )
 
 func TestCongressVoting(t *testing.T) {
 
 	var (
+		CongressAddr  = "GAIJAH4FCEB3AWS2NVKRNDGVJ5QPD7VNOQNGZEPL2FAI6GPZQNKEXRWI"
+		CongressSecret = "SBUONPFGBX7BF76ZZ2NOWWCJPG6NQQOQS37IL27JCUBPFAXFRYS3VDI2"
```

Comment:
This `CongreeAddr` is used also in `constant.go`. If this address is used without change.
This secret key can be known to all.
How about change the `CongreeAddr` and `CongressSecret` of test to another keypair?

Reply:
It is a client's testcode.
The Address should be the same as SEBAK's Congress Address.
When the SEBAK published, the address should be set by the environment variable.

Reply:
I mean that The string set in the constant.go is just default code.
It should be changed when launching the network. @soonkuk

Reply:
@kfangw Ok I got it.